core/src/main/scala/pureconfig/error/ConfigReaderFailures.scala

```diff
@@ -8,15 +8,15 @@ import scala.collection.mutable
 /**
  * A non-empty list of ConfigReader failures
  */
-case class ConfigReaderFailures(head: ConfigReaderFailure, tail: List[ConfigReaderFailure]) {
+case class ConfigReaderFailures(head: ConfigReaderFailure, tail: ConfigReaderFailure*) {
 
-  def toList: List[ConfigReaderFailure] = head +: tail
+  lazy val toList: List[ConfigReaderFailure] = head :: tail.toList
```

Comment:
Nit: why did you turn this into a `lazy val`? It seems that for our own use cases we always use it at most once to construct new `ConfigReaderFailures` and there's no expectation of it being memoized from the client code's perspective. Unless there's some good reason for it, `def`s are more appropriate to expose in APIs.

Reply:
Yes. This shouldn't have been made a `lazy val`. Reverted in 1536514.